### PR TITLE
Fixed state -> props mapping for login so it routes properly

### PR DESCRIPTION
**Pull request by htruong4**
Fixed the routing for logging in successfully from the login page. The mistake was grabbing a non existant property and this is my bad. Now props are succesfully passed down triggering a rerender.
Added proptypes to give an error if props are not passed down properly.

### CODE REVIEW

File: frontend/src/login/LoginPage.jsx [hunk 1]
```diff
@@ -88,7 +90,7 @@ class LoginComponent extends Component {
 
 const mapStateToProps = state => {
     return {
-        state: state.isLoggedIn
+        isValidated: state.isLoggedIn
```
Comment (forgeno) [inline frontend/src/login/LoginPage.jsx:92]: Should this be
isValidated: state.loginreducer.isloggedin?

File: frontend/src/login/LoginPage.jsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ class LoginComponent extends Component {
      * authentication
      */
     sendLoginRequest = () => {
+        console.log(this.props, "hello");
```
Comment (forgeno) [inline frontend/src/login/LoginPage.jsx]: Can you remove this comment?

**Review by forgeno — CHANGES_REQUESTED** [review]

File: frontend/src/login/LoginPage.jsx (hunk 1 above)
Comment (htruong1) [reply frontend/src/login/LoginPage.jsx]: ooof it looks like I didn't commit properly T_T hold on fam

Comment (htruong4) [reply frontend/src/login/LoginPage.jsx:92]: oh wait thats the wrong file

File: frontend/src/login/LoginPage.jsx (hunk 2 above)
Comment (shchau) [reply frontend/src/login/LoginPage.jsx]:
```suggestion
        console.log(this.props, "no");
```

**Pull request merged**